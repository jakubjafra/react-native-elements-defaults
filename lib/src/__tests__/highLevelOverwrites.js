const highLevelOverwrites = require('../highLevelOverwrites');

describe('highLevelOverwrites', () => {
    it('should contain all components', () => {
        expect(Object.keys(highLevelOverwrites)).toMatchSnapshot();
    });
});
