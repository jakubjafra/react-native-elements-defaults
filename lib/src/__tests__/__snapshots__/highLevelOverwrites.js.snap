// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`highLevelOverwrites should contain all components 1`] = `
Array [
  "AvatarOverwrite",
  "BadgeOverwrite",
  "ButtonOverwrite",
  "ButtonGroupOverwrite",
  "CardOverwrite",
  "CheckBoxOverwrite",
  "_ColorsOverwrite",
  "DividerOverwrite",
  "FeaturedTileOverwrite",
  "_FontsOverwrite",
  "FormInputOverwrite",
  "FormLabelOverwrite",
  "FormValidationMessageOverwrite",
  "HeaderOverwrite",
  "IconOverwrite",
  "ListOverwrite",
  "ListItemOverwrite",
  "PricingCardOverwrite",
  "RatingOverwrite",
  "SearchOverwrite",
  "SliderOverwrite",
  "SocialIconOverwrite",
  "TextOverwrite",
  "TileOverwrite",
]
`;
